### PR TITLE
Release mgr with aligned_free

### PR DESCRIPTION
**Pull request by Vt-Vrajendra**
mgr is allocated with _aligned_malloc and released with free instead of aligned_free

### CODE REVIEW

**Comment by gbtucker** [comment]
Thanks @Vt-Vrajendra.  Looks like there are two other cases of aligned alloc and free() in aes_thread.c.  Is there a way to test since if I remember aligned_free() is required by windows and the example uses pthreads?  Also remember to follow the [signoff](https://01.org/community/signed-process) process.

File: examples/saturation_test/md5_thread.c [hunk 1]
```diff
@@ -197,7 +197,7 @@ void *MB_THREAD_FUNC(void *arg)
 
       out:
 	free(ctxpool);
-	free(mgr);
+	aligned_free(mgr);
```
Comment (tkanteck) [inline examples/saturation_test/md5_thread.c:200]: Could you explain why this change is needed? From what I see in `man` use of `free()` is legitimate.
```
The  function  posix_memalign()  allocates  size bytes and places the address of the allocated memory in *memptr.  The address of the allocated memory will be a multiple of alignment, which must be a
power of two and a multiple of sizeof(void *). 
This address can later be successfully passed to free(3).
```

Comment (tkanteck) [reply examples/saturation_test/md5_thread.c:200]: Sorry, I missed the point it is for Windows. There are more changes needed here to make it compile on Windows, I am afraid. Only SHA1 and GCM examples compile on Windows.

**Comment by pablodelara** [comment]
Given there are no responses, I will close this PR and we'll take this work from here.
